waspc/data/Generator/templates/server/src/routes/auth/logout.ts

```diff
@@ -0,0 +1,14 @@
+import { handleRejection } from '../../utils.js'
+import { throwInvalidCredentialsError } from '../../auth/utils.js'
+import { invalidateSession } from '../../auth/session.js'
+
+export default handleRejection(async (req, res) => {
+  if (req.sessionId) {
+    await invalidateSession(req.sessionId)
+    delete req.sessionId
+    delete req.user
```

Comment:
I am curious, why is this important?

Reply:
Good question, I wanted to do it as "better safe than sorry" move. But `req` is valid for only one request, so there is not real need for this **unless** some other middleware is executed after this handler.

We don't currently have anything happening after the request handler, so I'll simplify the code.

Reply:
No worries, you can leave it if you like it, just to be safe as you said.